docs/source/notes/cuda.rst

```diff
@@ -131,6 +131,47 @@ For more information about TF32, see:
 .. _CUDA 11: https://devblogs.nvidia.com/cuda-11-features-revealed/
 .. _Ampere architecture: https://devblogs.nvidia.com/nvidia-ampere-architecture-in-depth/
 
+.. _fp16reducedprecision:
+
+Reduced Precision Reduction in FP16 GEMMs
+-----------------------------------------
+
+fp16 GEMMs are potentially done with reduced precision reductions (e.g., in fp16 rather than fp32). This reduction in precision can allow for higher performance on certain workloads (particularly those with a large `k` dimension) and GPU architectures at the cost of numerical precision and potential for overflow.
+
+Some example benchmark data on V100
+.. code::
```

Comment:
[nit-picking] The table rendered by `torch.utils.benchmark` seems compatible with Sphinx, which obviates this `.. code::` directive.
![Screenshot from 2021-11-06 15-09-06](https://user-images.githubusercontent.com/16191443/140599886-c23d0706-3f96-4239-b6da-a8223a8bfa19.png)

Or fix the indentation of this table including "(time ...)".

Reply:
Fix this please